profile_specs/monitor_profiles.json

```diff
@@ -1,48 +1,11 @@
 [
   {
-    "name": "Profile_2015_04_01",
-    "resourceTypes": {
-      "Microsoft.Monitor" : {
-        "2015-04-01": ["*"]
-      }
-    },
-    "output_dir": "../../../management/azure_mgmt_monitor/lib/profiles",
-    "individual_gem_profile": true
-  },
-  {
-    "name": "Profile_2016_03_01",
-    "resourceTypes": {
-      "Microsoft.Monitor" : {
-        "2016-03-01": ["*"]
-      }
-    },
-    "output_dir": "../../../management/azure_mgmt_monitor/lib/profiles",
-    "individual_gem_profile": true
-  },
-  {
-    "name": "Profile_2016_09_01",
-    "resourceTypes": {
-      "Microsoft.Monitor" : {
-        "2016-09-01": ["*"]
-      }
-    },
-    "output_dir": "../../../management/azure_mgmt_monitor/lib/profiles",
-    "individual_gem_profile": true
-  },
-  {
     "name": "Profile_2017_04_01",
     "resourceTypes": {
       "Microsoft.Monitor" : {
-        "2017-04-01": ["*"]
-      }
-    },
-    "output_dir": "../../../management/azure_mgmt_monitor/lib/profiles",
-    "individual_gem_profile": true
-  },
-  {
-    "name": "Profile_2017_05_01_Preview",
-    "resourceTypes": {
-      "Microsoft.Monitor" : {
+        "2015-04-01": ["*"],
```

Comment:
if we need to pick specific specs, how we would do that? I think we could go with this approach for now, but it's actually not safe in my opinion, to include everything under an api-version, packages/tags may pick specific files.